src/opm/parser/eclipse/EclipseState/Schedule/eval_uda.cpp

```diff
@@ -65,7 +65,7 @@ double eval_group_uda(const UDAValue& value, const std::string& group, const Sum
         output_value = st.get(string_var);
 
     // We do not handle negative rates.
-    output_value = std::max(0.0, output_value);
+    output_value = std::max(1.0e-20, output_value);
```

Comment:
Maybe add a comment that these very small nonzero values are there for a reason.